graphics/timg/Portfile

```diff
@@ -3,6 +3,10 @@
 PortSystem          1.0
 PortGroup           cmake 1.1
 PortGroup           github  1.0
+PortGroup           legacysupport 1.1
+
+# memmem, filesystem
+legacysupport.newest_darwin_requires_legacy 18
```

Comment:
You forgot set ```legacysupport.use_mp_libcxx``` to ```yes```.

Reply:
Done, also changed commit title to be more accurate.